discovery-provider/src/queries/health_check.py

```diff
@@ -36,24 +37,8 @@ def _get_db_block_state(latest_blocknum, latest_blockhash):
         # Fetch latest block from DB
         db_block_query = session.query(Block).filter(Block.is_current == True).all()
         assert len(db_block_query) == 1, "Expected SINGLE row marked as current"
+        return helpers.model_to_dictionary(db_block_query[0])
 
-        health_results = {
-            "web": {
-                "blocknumber": latest_blocknum,
-                "blockhash": latest_blockhash,
-            },
-            "db": helpers.model_to_dictionary(db_block_query[0]),
-            "git": os.getenv("GIT_SHA"),
-        }
-
-        block_difference = abs(
```

Comment:
because this no longer exists block_check breaks


```python3
[2020-08-04 01:04:57,216] {src.exceptions:35} (ERROR) - Non Audius-derived exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/audius-discovery-provider/src/queries/health_check.py", line 162, in block_check
    if health_results["block_difference"] > healthy_block_diff:
KeyError: 'block_difference'
```

Reply:
sorry -- I was super unclear here. The issue that was broken was the block check in the original PR?

Reply:
I'm going to rework this PR and add some unit tests

Reply:
@raymondjacobson Sorry that's my bad. I should have been more clear that there were two issues with this. Pre QA Joey fixed the redis set but then when we discovered we'd need to rework block check, we instead decided to revert the pr

Reply:
Yeah -- i gotcha now. I didn't understand that there were two issues. I think the answer is we just need testing/typing that can catch things like missing variables like this.